### DISCUSSION TITLE
SET_DATE

### DISCUSSION THREAD

**Discussion by Whitling2k** [MMRIZE/MMM-CalendarExt3#57] [General]
Is there a simple way to make CX3_SET_DATE jump to the current date?

**Comment by eouia** [comment]
Unfortunately, nothing now. But I realized it is not so convenient, so I will change in some days. (I have not enough time at this moment)

**Reply by Whitling2k** [reply]
Works great! Thanks :-)

**Comment by KristjanESPERANTO** [comment]
This feature has been implemented! :slightly_smiling_face: 

The old `CX3_SET_DATE` notification is now deprecated and has been replaced by a more flexible system:

**To jump back to the current date:**

```javascript
// Option 1: Set referenceDate to null (current date)
this.sendNotification("CX3_SET_CONFIG", {
  referenceDate: null
});

// Option 2: Reset to original configuration
this.sendNotification("CX3_RESET");
```

The new `CX3_SET_CONFIG` notification is much more powerful - you can change any configuration property, not just the date. More details can be found in the [README](https://github.com/MMRIZE/MMM-CalendarExt3#notification).

Closing this as resolved.
